docs/source/create_channel/create_channel_participation.md

```diff
@@ -4,19 +4,21 @@ Fabric v2.3 introduces the ability to create channels between network members wi
+ * **Simplified ordering node creation process**: Because a genesis block no longer needs to exist before an ordering node is created, admins can now focus on the process of setting up their infrastructure and making sure their node is working correctly before joining a particular application channel. Additionally, because an ordering node no longer needs to scan the system channel to determine which channels it is joined to and then replicate those ledgers, the node starts up more rapidly.
 
 **Benefits of the new process:**
 
- * **Increased Privacy** - A single orderer does not know about all the channels on the ordering service.
- * **Scalability** - When there is a large number of ordering service nodes (orderers) on the system channel, starting the orderer can take a long time as it has to wait for each orderer to replicate the channel ledger.
- * **Operational benefits** - Updating an application channel no longer requires write permissions from a separate ordering organization. Also, you now have a way to list all the channels an orderer is a member of. Lastly, you have the ability to easily remove a channel and its respective resources from an orderer.
+ * **Increased privacy**: Because all ordering nodes used to be joined to the system channel, every ordering node in a network knew about the existence of every channel on that ordering service, even if the node itself wasn't a consenter on that channel and therefore wasn't ordering or store its blocks. Now, an ordering node only knows about the channels it is joined to.
+ * **Scalability**: When there is a large number of ordering nodes on the system channel, starting the node can take a long time as it has to wait for each orderer to replicate the system channel ledger showing it has joined. Additionally, large numbers of nodes on the system channel increases the number of "alive" messages being sent between nodes. In large system channels with large numbers of nodes, this can cause processing issues, even if no application channels has a large number of ordering nodes on it.
+ * **Flexibility**: Whereas ordering nodes could only be part of a single ordering service, orderers can now join or leave channels as needed, similar to how peers can join any channel its organization is a member of.
+ * **Operational benefits**:
+  * It is now easy to list the channels the ordering node is a consenter on.
+  * Is now easy to delete a channel and all of the ledger associated with that channel.
```

Comment:
Consistency with the bullet above: `It is now easy`

Reply:
Done